### ISSUE TITLE
Make docs feedback widget more discoverable and engaging

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#6713]
It's current at the bottom of every page. Users who don't make it to the bottom of course probably don't offer feedback. How can we increase awareness and engagement. 

One idea seen in lots of other docs is to move to the top of page and/or make it a 5-star-type rating.

**Examples**

[Google](https://stripe.com/docs/payments/accept-a-payment#ios):
<img width="1018" alt="Screen Shot 2020-02-28 at 8 21 16 AM" src="https://user-images.githubusercontent.com/15893490/75551975-5115b400-5a03-11ea-8720-62148d6833bc.png">

[Twilio](https://stripe.com/docs/payments/accept-a-payment#ios):
<img width="1045" alt="Screen Shot 2020-02-28 at 8 22 08 AM" src="https://user-images.githubusercontent.com/15893490/75552045-71de0980-5a03-11ea-8d0d-bc103d9abb37.png">

**Comment by jseldess** [comment]
Another idea is to pin it to the lower right of the screen like in Mongo's docs:

<img width="1405" alt="Screen Shot 2020-04-02 at 12 13 14 AM" src="https://user-images.githubusercontent.com/15893490/78210017-d236ef80-7476-11ea-97b5-eab99da780bc.png">

**Comment by luciaaaaaaa** [comment]
Not sure about this one. This widget is not relevant to the user unless they've read the entire article, and it might distract from our goal of making it very easy for them to find relevant articles. Is the response rate really low? how does the response rate of this widget vs the CSAT score compare?

**Comment by jseldess** [comment]
Sorry for the late response. It's not true that the feedback widget is only relevant if the user has read the entire article. For a tutorial that requires a user to move from start to finish that might make sense. But not for reference pages or even how tos where users may find specific details or guidance on the page. 

In any case, since opening this issue, we've stopped using the feedback widget for calculating CSAT; we have a separate survey for that. Closing this.
